Storage/Sources/StorageConfig.swift

```diff
@@ -12,11 +12,13 @@ public struct GoogleCloudStorageConfiguration: GoogleCloudAPIConfiguration {
     public var scope: [GoogleCloudAPIScope]
     public let serviceAccount: String
     public let project: String?
+    public let subscription: String?
     
     public init(scope: [GoogleCloudStorageScope], serviceAccount: String, project: String?) {
         self.scope = scope
         self.serviceAccount = serviceAccount
         self.project = project
+        self.subscription = nil
```

Comment:
We shouldn't need this in the initializer right? It can be initialized to nil on the property itself. Other than that everything looks good.

Reply:
Yeah, dont need it for StorageApi